base/applications/network/telnet/src/tnerror.cpp

```diff
@@ -91,7 +132,23 @@ int printm(LPTSTR szModule, BOOL fSystem, DWORD dwMessageId, ...)
 
 	va_list Ellipsis;
 	va_start(Ellipsis, dwMessageId);
+#ifdef __REACTOS__
+	LPWSTR wszMessage = NULL;
```

Comment:
Tip: use the original name `pszMessage` here, so that when freeing the allocated buffer below....